### PR TITLE
Allow Non-breaking Spaces

### PR DESCRIPTION
**Pull request by tariq-dt**
# Why

If the value string had non-breaking spaces ('\u00A0'), the indentation would not work. 

# What changed

A non-breaking space character is treated same as the space character

# Test plan

Test cases can be added which have non-breaking spaces as well as ordinary spaces in the input

### CODE REVIEW

**Comment by sergeichestakov** [comment]
sounds reasonable cc @Monkatraz can you tal?

**Comment by MrRobz** [comment]
Also needed the same fix. Can we merge this @sergeichestakov @Monkatraz ?

**Comment by sergeichestakov** [comment]
I'm in favor, think this change looks good, but unfortunately I left the org and no longer have write access to merge so will leave it up to @Monkatraz cc also @masad-frost

**Comment by masad-frost** [comment]
Thanks! Yeah we'll merge and mint a new release soon.

**Review by Monkatraz — APPROVED** [review]

**Comment by Monkatraz** [comment]
Published 6.5.3

**Pull request merged**
